symbulate/random_variables.py

```diff
@@ -254,10 +259,16 @@ def __init__(self, random_variable, condition_event):
         
     def draw(self):
         """
-            A function, that takes no arguments, explaining how to draw one 
-        outcome from the Probability Space described by the random variable.
+        A function that takes no arguments and returns a single set 
+            of values from the probability spaces defined in the conditional 
```

Comment:
Again, you're not returning values from the probability space. You're returning values of the random variable. Please rewrite this.